sdk/face/ai-vision-face-rest/test/public/longRunningOperation.spec.ts

```diff
@@ -14,114 +14,158 @@ const randomUUID = (): string =>
   );
 
 describe("PersonDirectory", () => {
-    let recorder: Recorder;
-    let client: FaceClient;
-  
-    beforeEach(async (context) => {
-      recorder = await createRecorder(context);
-      client = await createClient(recorder);
+  let recorder: Recorder;
+  let client: FaceClient;
+
+  beforeEach(async (context) => {
+    recorder = await createRecorder(context);
+    client = await createClient(recorder);
+  });
+
+  afterEach(async () => {
+    if (recorder?.recordingId) {
+      await recorder.stop();
+    }
+  });
+
+  it("CreateDynamicPersonGroup", async () => {
+    const dynamicPersonGroupId = recorder.variable("dynamicPersonGroupId", randomUUID());
+    const createDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .put({
+        body: { name: dynamicPersonGroupId },
+      });
+    assert.equal(createDynamicPersonGroupResponse.status, "200");
+
+    const deleteDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .delete();
+    const deleteDynamicPersonGroupPoller = await getLongRunningPoller(
+      client,
+      deleteDynamicPersonGroupResponse,
+    );
+    await deleteDynamicPersonGroupPoller.pollUntilDone();
+    assert.equal(deleteDynamicPersonGroupPoller.getOperationState().status, "succeeded");
+  });
+
+  it("CreateDynamicPersonGroupWithPerson", async () => {
+    const createPersonResponse = await client.path("/persons").post({
+      body: { name: "test" },
     });
-  
-    afterEach(async () => {
-      if (recorder?.recordingId) {
-        await recorder.stop();
-      }
+    if (isUnexpected(createPersonResponse)) {
+      throw new Error(createPersonResponse.body.error.message);
+    }
+    const { personId } = createPersonResponse.body;
+
+    const dynamicPersonGroupId = recorder.variable("dynamicPersonGroupId", randomUUID());
+    const createDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .put({
+        body: {
+          name: dynamicPersonGroupId,
+          addPersonIds: [personId],
```

Comment:
Cool! It seems the type system could successfully overload the type.

Reply:
Yeah it works well 😄